front/lib/production_checks/checks/managed_data_source_gdrive_gc.ts

```diff
@@ -27,7 +27,7 @@ export const managedDataSourceGCGdriveCheck: CheckFunction = async (
 
     // Retrieve all documents from the connector (first)
     const connectorDocuments: { id: number; coreDocumentId: string }[] =
-      await connectorsReplica.query(
+      await connectorsDb.query(
```

Comment:
This one can be a bit taxing for the primary no?... Do we use the primary or the secondary for `getCoreDocuments`

Reply:
that one is quite small, but for the getCoreDocuments we are iterating on every data_source, and it can return a lot of docs - so yes running this one on  primary db will  be more costly. We can keep the whole check on replica to avoid loading the db , wdyt ?

Reply:
Yes bullish on keeping this one on secondary as much as possible

Reply:
👍 
keeping managed_data_source_gdrive_gc and getCoreDocuments on replica